src/Renci.SshNet/SubsystemSession.cs

```diff
@@ -72,6 +72,14 @@ public bool IsOpen
             get { return _channel is not null && _channel.IsOpen; }
         }
 
+        public ILoggerFactory SessionLoggerFactory
+        {
+            get
+            {
+                return _session.SessionLoggerFactory;
```

Comment:
_session could potentially be null here. I think it is fine to make _session readonly and remove the `_session = null;` line in Dispose